recipe/meta.yaml

```diff
@@ -73,39 +80,30 @@ requirements:
     - libboost-devel
     - pyyaml
     - ply
+    - libpisp        # [variant == "rpi_fork"]
+    # libpisp build dependency
+    - nlohmann_json  # [variant == "rpi_fork"]
   run:
     - python
 test:
   commands:
+    - cam -l
     - test -f $PREFIX/lib/libcamera.so
     - test -f $PREFIX/include/libcamera/libcamera/libcamera.h
+    # command only available in the rpi fork
+    - libcamerify -h  # [variant == "rpi_fork"]
```

Comment:
I think this is false, and only true because you had enabled a v4l2 flag only on rpi. 

The flag should just be enabled on both variants, there is no need to enable it only on one.